### PR TITLE
fix minor formatting and grammar issues in docs

### PR DESCRIPTION
**Pull request by jameslamb**
Fixes some minor formatting and grammar issues I noticed while reading through this project's documentation.

## Changes

- fixes formatting in `.md` files
- fixes small grammar issues

## Testing

None

## Notes

None

## Checklist

- [x] PR has an informative and human-readable title (this will be pulled into the release notes)
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code can be automatically merged (no conflicts)
- [x] Code passed the pre-commit check & code is left cleaner/nicer than when first encountered.
- [ ] Passes all existing automated tests
- [x] Any _change_ in functionality is tested
- [x] New functions are documented (with a description, list of inputs, and expected output)
- [x] Placeholder code is flagged / future TODOs are captured in comments
- [x] Project documentation has been updated if adding/changing functionality.
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Python - local testing

- [ ] python 3.6
- [ ] python 3.7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ In general we prescribe the following:
 4. Write a simple script so that you can easily run things end to end.
 5. Join our [discord](https://discord.gg/wCqxqBqn73) community to chat/ask Qs/etc.
 
-For the backstory on Hamilton we invite you to watch ~9 minute lightning talk on it that we gave at the apply conference:
+For the backstory on Hamilton we invite you to watch a roughly-9 minute lightning talk on it that we gave at the apply conference:
```
Comment (jameslamb) [inline README.md:153]: `~` is a special markdown character used for strike-through text: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#styling-text

File: basics.md [hunk 2]
```diff
@@ -118,7 +118,7 @@ Otherwise the contents of the _config_ dictionary should include all the inputs
 want to create. The configuration dictionary should not be used for overriding what Hamilton will compute.
 To do this, use the `override` parameter as part of the `execute()` -- see below.
 
-#### *modules: ModuleType
+#### modules: ModuleType
```
Comment (jameslamb) [inline basics.md:121]: based on the formatting of other sections at https://github.com/stitchfix/hamilton/blob/49f11ede9a79478f31b8ded2673ec0f977033562/basics.md, I think this `*` was unintended.

Screen Shot 2022-03-27 at 4 30 04 PM

File: decorators.md [hunk 3]
```diff
@@ -121,7 +121,7 @@ def copy_of_x(x: pd.Series) -> pd.Series:
 ```
 The example here is a function, that all that it does, is sum all the parameters together. So we can annotate it with
 the `@does` decorator and pass it the `sum_series` function.
-The `@does` decorator is currently limited to just allow functions that consist only of one argument, a generic **kwargs.
+The `@does` decorator is currently limited to just allow functions that consist only of one argument, a generic `**kwargs`.
```
Comment (jameslamb) [inline decorators.md:124]: Markdown renderers recognize `**` as "bold": https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#styling-text

But I think here the intention was to refer to the literal Python code `**kwargs`.

**Comment by elijahbenizzy** [comment]
Nice finds! I think the only intentional one is the *modules :)

**Comment by jameslamb** [comment]
> I think the only intentional one is the *modules

Ok, sure. Did you mean that you want a literal `*` at the beginning of that section, or that it was intended to be closed with another `*` to italicize it?

**Review by elijahbenizzy — CHANGES_REQUESTED** [review]

**Comment by skrawcz** [comment]
> > I think the only intentional one is the *modules
> 
> Ok, sure. Did you mean that you want a literal `*` at the beginning of that section, or that it was intended to be closed with another `*` to italicize it?

We meant for it to be a literal `*` to match the parameter declaration.

File: basics.md (hunk 2 above)
Comment (jameslamb) [reply basics.md]: ahhh got it! Pushed https://github.com/stitchfix/hamilton/pull/102/commits/dc920431b2db796b6da86842349ed465ce621b30 adding it back, with escaping.

You can see how GitHub renders it on my branch.

https://github.com/jameslamb/hamilton/blob/docs-fixes/basics.md

Screen Shot 2022-03-30 at 7 23 38 PM

**Review by skrawcz — APPROVED** [review]

**Comment by skrawcz** [comment]
@jameslamb mind if I squash merge? I want to remove the merge commit - else I would do a rebase merge.

**Pull request merged**